libyul/backends/wasm/polyfill/Memory.yul

```diff
@@ -61,6 +61,15 @@ function memoryguard(x:i64) -> y1, y2, y3, y4 {
 	y4 := x
 }
 
+function linkersymbol(x) -> y1, y2, y3, y4 {
+}
```

Comment:
I would put an `invalid()` into these. And likely wouldn't put them into `Memory.yul`, if there's a better place. However not including them here is also good, we just won't enable these tests?

Reply:
ok, moved them to `Interface.yul`